README.md

```diff
@@ -1,186 +1,80 @@
-# mock-spacestation
```

Comment:
nit: throughout this documentation we're using `Spacestation`, `spacestation`, `SpaceStation`, `spaceStation`, `Space station`, and `space station` interchangeably and seemingly the same for groundstations.

Let's settle on one convention and use the same convention for the groundstation.